docs/snippets/lpt_intel_cpu_plugin.cpp

```diff
@@ -70,29 +70,29 @@ manager.run_passes(nGraphFunc);
 using namespace ngraph::pass::low_precision;
 if (useLpt) {
     // Low precision transformations plugin specific configuration: restrictions definition
-    auto supportedPrecisions = std::vector<OperationPrecisionRestriction>({
-        OperationPrecisionRestriction::create<ngraph::opset1::Convolution>({
+    auto supportedPrecisions = std::vector<PrecisionsRestriction>({
+        PrecisionsRestriction::create<ngraph::opset1::Convolution>({
             {0, {ngraph::element::u8}},
             {1, {ngraph::element::i8}},
         }),
-        OperationPrecisionRestriction::create<ngraph::opset1::ConvolutionBackpropData>({
+        PrecisionsRestriction::create<ngraph::opset1::ConvolutionBackpropData>({
             {0, {ngraph::element::u8, ngraph::element::i8}},
             {1, {ngraph::element::i8}}
         }),
-        OperationPrecisionRestriction::create<ngraph::opset1::GroupConvolution>({
+        PrecisionsRestriction::create<ngraph::opset1::GroupConvolution>({
             {0, {ngraph::element::u8}},
             {1, {ngraph::element::i8}}
         }),
-        OperationPrecisionRestriction::create<ngraph::opset1::Multiply>({
+        PrecisionsRestriction::create<ngraph::opset1::Multiply>({
             {0, {ngraph::element::u8}},
             {1, {ngraph::element::i8}},
         }),
     });
 
     // Low precision transformations plugin specific configuration: per-tensor quantization operations definition
-    auto perTensorQuantization = std::vector<OperationPerTensorQuantizationRestriction>({
-        OperationPerTensorQuantizationRestriction::create<ngraph::opset1::Convolution>({0}),
-        OperationPerTensorQuantizationRestriction::create<ngraph::opset1::ConvolutionBackpropData>({0})
+    auto perTensorQuantization = std::vector<QuantizationGranularityRestriction>({
+        QuantizationGranularityRestriction::create<ngraph::opset1::Convolution>({0}),
```

Comment:
Since it is anyway quantization related stuff you can call it just `GranularityRestriction`.